Source/WebKit/UIProcess/Cocoa/PageClientImplCocoa.mm

```diff
@@ -299,12 +299,12 @@
 
 void PageClientImplCocoa::unifiedTextReplacementActiveWillChange()
 {
-    [m_webView willChangeValueForKey:unifiedTextReplacementActiveKey];
+    [m_webView willChangeValueForKey:@"writingToolsActive"];
```

Comment:
Can this be a static define somewhere? Like I assume unifiedTextReplacementActiveKey was?

Reply:
+1